src/js/components/Anchor/StyledAnchor.js

```diff
@@ -71,9 +72,13 @@ const StyledAnchor = styled.a.withConfig(styledComponentsConfig)`
   ${(props) =>
     props.hasIcon &&
     !props.hasLabel &&
-    `
-    padding: ${props.theme.global.edgeSize.small};
-  `}
+    edgeStyle(
+      'padding',
+      props.theme.anchor.iconOnly.pad,
+      props.responsive,
```

Comment:
It seems like this wasn't responsive previously, so we should probably just set to false?
```suggestion
      false,
```

Reply:
yes I was going to ask you your thoughts